src/AdminApi/Redirect/Builders/GetRedirectsRequest.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace Yaspa\AdminApi\Redirect\Builders;
+
+use Yaspa\Constants\RequestBuilder;
+use Yaspa\Interfaces\PagingRequestBuilderInterface;
+use Yaspa\Traits\AuthorizedRequestBuilderTrait;
+
+/**
+ * Class GetRedirectsRequest
+ *
+ * @package Yaspa\AdminApi\Redirect\Builders
+ * @see https://help.shopify.com/api/reference/redirect#index
+ */
+class GetRedirectsRequest implements PagingRequestBuilderInterface
+{
+    use AuthorizedRequestBuilderTrait;
+
+    const URI_TEMPLATE = 'https://%s.myshopify.com/admin/redirects.json';
+
+    /** @var array|int[] $ids */
+    protected $ids;
```

Comment:
The `id`'s parameter isn't documented on Shopify and testing came back negative.

Reply:
Fixed in commit 5ebffee